contentElements/MultiLineInputElement.js

```diff
@@ -92,14 +103,53 @@ MultiLineInputElement.prototype.reAddEntities = function (entitiesArr) {
             entitiesArr.push(this.variable());
         }
     }
-
+    this.modifier().reAddEntities(entitiesArr);
     this.questionText().reAddEntities(entitiesArr);
 };
 
 MultiLineInputElement.prototype.selectTrialType = function (selectionSpec) {
+    this.modifier().selectTrialType(selectionSpec);
     this.questionText().selectTrialType(selectionSpec);
 };
 
+MultiLineInputElement.prototype.dispose = function () {
```

Comment:
All these 4 function from here down are doubled in the file now. Please remove the doubled code blocks again. In general, merge requests should really only contain the code changes that are required for the new feature. So you should first check through the change list and make sure that there is nothing in there, that you did not actually want to change.

Reply:
I removed the mentioned functions